pkg/util/objtools/s3.go

```diff
@@ -13,11 +13,13 @@ import (
 )
 
 type S3ClientConfig struct {
-	BucketName      string
-	Endpoint        string
-	AccessKeyID     string
-	SecretAccessKey string
-	Secure          bool
+	BucketName       string
+	Endpoint         string
+	AccessKeyID      string
+	SecretAccessKey  string
+	Secure           bool
+	DisableMultipart bool
```

Comment:
These options shouldn't be added to the client in `pkg/util/objtools`. It doesn't use `objstore` and intentionally doesn't support a wider set of configuration currently.

Reply:
OK I'll remove the a60a29aac commit then